hq/README.md

```diff
@@ -56,6 +56,18 @@ AWS_ACCOUNTS=[
 ]
 ```
 
+### Required AWS Security Policies
```

Comment:
While this is a very useful addition, would it not be better to point the developer at the actual policy (in cloudformation)? As it is, we now have two places that need to be kept up to date.

Speaking of, this permissions change needs to be made to the CloudFormation.
https://github.com/guardian/security-hq/blob/master/cloudformation/watched-account.template.yaml

Reply:
(This one's a blocker because Security-HQ won't work if these permissions aren't set)

Reply:
will update one, PR left old after moving to CF template.